test/crawler/keyword/keyword_extractor_test.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Crawler.Keyword.KeywordExtractorTest do
+  use CrawlerWeb.ConnCase, async: true
+
+  alias Crawler.Google.Client, as: GoogleClient
+  alias Crawler.Keyword.KeywordExtractor
+
+  describe "parse/1" do
+    test "given HTML, returns parsed results" do
```

Comment:
and add negative scenario test here (related to my comment about parsing errors)